pkg/controller/actuator.go

```diff
@@ -46,6 +47,18 @@ import (
 // ActuatorName is the name of the Certificate Service actuator.
 const ActuatorName = "shoot-cert-service-actuator"
 
+var (
+	scheme  *runtime.Scheme
+	decoder runtime.Decoder
+)
+
+func init() {
```

Comment:
Can you explain why you added this init func instead of letting the scheme being inject by the Controller-Runtime?

Reply:
I introduced this, after decoding did not work for CertConfig. But probably the real issue was the missing call to add the service schema with `install.Install(scheme)`. I will revert and check if it works when added to the init func in pkg/cmd/options.go.

Reply:
Okay, thank you 👍